hy/MaybeHug.hy

```diff
@@ -11,10 +11,10 @@
   [[hug (fn [self]
     (print "HUG!"))]])
 
-(defclass Empathic [MaybeHug] [])
+(defclass Emphatic [MaybeHug] [])
```

Comment:
This spelling was actually correct! No need for this patch: "emphatic" means "with emphasis", and "empathic" means "with empathy", which is what we're going for :heart: